dipp/tensorflow/layers.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2014-2017 The DIPP contributors
+#
+# This file is part of DIPP.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+import tensorflow as tf
```

Comment:
Commenting here. In general, at least a minimal docstring that says what the function/class does would be great.